lib/utils/conf_gen.py

```diff
@@ -959,6 +984,28 @@ async def _generate_intermediate(
         await pipeline(context_dict)
         return context_dict[INTERMEDIATE]
 
+    def _sort_keys(self, intermediate_dict: dict[IntermediateKey | str, Any]):
+        str_keys = []
+        list_keys = []
+        unnamed_keys = []
+        named_keys = []
+
+        for key in intermediate_dict:
+            if isinstance(key, str):
+                str_keys.append(key)
+            elif isinstance(key, InterListKey):
+                list_keys.append(key)
+            elif isinstance(key, InterUnnamedSectionKey):
+                unnamed_keys.append(key)
+            elif isinstance(key, InterNamedSectionKey):
+                named_keys.append(key)
+
+        str_keys.sort()
+        list_keys.sort(key=lambda x: x.name)
+        named_keys.sort(key=lambda x: x.type + x.name)
```

Comment:
Why no sort for unnamed_keys?

Reply:
I suppose I could. I just did not see it as necessary like the others. The reason I did this is because when requesting dc and ns configs they are called concurrently and the results could be in any order. Dicts in python iterate by insertion order. The responses that provide the unamed sections are always in the same order and come from a single info request. 

This was mostly done to make comparisons easier in testing